syntax = "proto3";

import "google/api/annotations.proto";

option go_package="gitlab.sz.lowcost.com/vip/api/helloworld";

package lowcost.vip.helloworld;

// The greeting service definition.
service Greeter 
{
  // Sends a greeting
    rpc SayHello (HelloRequest) returns (HelloReply);
}

// The request message containing the user's name.
message HelloRequest 
{
    string name = 1;
}


// The response message containing the greetings
message HelloReply 
{
    string message = 1;
}
